src/celeritas/user/StepInterface.hh

```diff
@@ -16,6 +18,16 @@ namespace celeritas
 //---------------------------------------------------------------------------//
 /*!
  * Callback class to gather and process data from many tracks at a single step.
+ *
+ * The filtering mechanism allows different step interfaces to gather data from
+ * different detector volumes. Filtered step interfaces cannot be combined with
+ * unfiltered in a single hit collector. (FIXME: maybe we need a slightly
+ * different class hierarchy for the two cases?) If detectors are in use, and
```

Comment:
What will happen if a user tries to combine step interfaces with different modes?

Reply:
I'm not sure whether to call it "unspecified" or to raise an error or what... maybe raise an error for now to be on the safe side.